DuckDuckGo/AppDelegate.swift

```diff
@@ -45,6 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         startMigration()
+        TrackerLoader.shared.updateTrackers()
```

Comment:
Note that this causes the trackers request to happen every time the application becomes active.  That means, for instance, if the user swipes up/away the control panel or notifications.  

That said, it looks like Alamofire uses the default URLCache configuration, but it's not entirely clear to me what that is.  Do we need to be more explicit about when the tracker data is downloaded to prevent too much data usage or, conversely, getting out of date by using a cached version?